### PR TITLE
Fix syntax error and update example on homepage

### PR DESCRIPTION
**Pull request by wezm**
I tried building the code on the homepage and ran into some issues. This PR updates the example code so that it compiles and runs.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -62,12 +68,13 @@ Create an application, etc.
...
 fn main() {
-    let application = Application::new("com.github.gtk-rs.examples.basic", Default::default()))
+    let application = Application::new("com.github.gtk-rs.examples.basic", Default::default())
         .expect("failed to initialize GTK application");
 
     application.connect_activate(|app| {
-        let window = ApplicationWindow::new();
+        let window = ApplicationWindow::new(app);
         window.set_title("First GTK+ Program");
+        window.set_border_width(10);
```
Comment (GuillaumeGomez) [inline index.md]: I'm not sure this line is very useful in this example. ;°

Comment (wezm) [reply index.md]: Perhaps not but I was mostly just updating it to do what https://github.com/gtk-rs/examples/blob/master/src/bin/basic.rs does.

Comment (GuillaumeGomez) [reply index.md]: The goal is to remain as small as possible in here to just let users see a sample. Please remove this line and then I merge.

Comment (wezm) [reply index.md]: Fixed :)

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**